.pre-commit-config.yaml

```diff
@@ -7,27 +7,13 @@ ci:
 
 repos:
 -   repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: 'v0.12.2'
+    rev: 'v0.12.7'
     hooks:
+    # Run the linter:
```

Comment:
I think this should be `id: ruff-check` nhow, see https://github.com/astral-sh/ruff-pre-commit. Caught a warning about that in a different project